woocommerce/woocommerce
# WooCommerce Utilities CLI: Add banner

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This adds a banner when using the new WooCommerce utilities CLI.

<img width="814" alt="image" src="https://user-images.githubusercontent.com/1922453/233496057-3cd80bce-a75b-4407-8688-fb8efbe8f386.png">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `pnpm install`
2. Run a command such as `pnpm run utils code-freeze verify-day`
3. See the banner, make sure it makes sense. Did we name this utility correctly?
4. Does `pnpm run utils` makes sense as a name to invoke the commands intended?
